Appendix pachinko results (#45)
MrLongbottom/P10-Article: sections/appendix/pachinko_results.tex

\subsection{Pachinko Results}\label{app:pachinko_res}

\autoref{tab:pachinko_topics} shows the final lowest level topics of the pachinko model.

> Note that while most of the topics are semantically coherent, there are a some topics (eg. topic 19 and 42) which consist entirely of words which provide little context or semantic meaning.

Reviewer: ```suggestion
Note that while most of the topics are semantically coherent, there are some topics (eg. topic 19 and 42) which consist entirely of words which provide little context or semantic meaning.
```

> The model has also learned to group words that do not belong into any good topics.

Reviewer: ```suggestion
The model has also learned to group words that do not belong to any good topics.
```

> This is a good feature which allows the model to apply an extra layer of preprocessing, automatically filtering away irrellevant words into topics.

Reviewer: ```suggestion
This is a good feature that allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics.
```

> This is a good feature which allows the model to apply an extra layer of preprocessing, automatically filtering away irrellevant words into topics.

Author: ```suggestion
This is a good feature which allows the model to apply an extra layer of preprocessing, automatically filtering away irrellevant words into topics.
\todo[inline]{ref to hlda and possibly one newer paper (simba knew one) that does the same thing.}
```

		83 & ord, bogen, bog, liv, verden, skrevet, hendes, du, historie, skriver \\
		84 & løbet, rebild, blokhus, løb, deltagerne, deltagere, turen, kl, kilometer, tur \\
		85 & thisted, thy, mors, hanstholm, klitmøller, vorupør, hurup, lokale, nationalpark, morsø \\
		86 & handbold, mors, thy, hold, kamp, sæson, kampe, kampen, point, holdet \\
		87 & prins, fylder, henrik, hans, larsen, tv, kim, københavn, senere, født \\
		88 & eu, britiske, brexit, storbritannien, aftale, london, may, premierminister, johnson, theresa \\
		89 & frivillige, foreningen, løgstør, aktiviteter, lokale, foreninger, medlemmer, forening, formand, lørdag \\
		90 & landbrug, landbruget, landmænd, vand, miljø, affald, bedre, natur, vandløb, fødevarer \\
\end{longtable}

\autoref{tab:pachinko_mid_topics} gives an overview of how the taxonomy topics in the third layer of the pachinko model, are connected to the fourth layer topics that were generated by the model.
Some of these connections make a lot of sense, such as the 'Økonomi' (Economy) topic which has the three filler topics: 79, 75, and 42, which consists of words with little semantic value, and two topics which are about money: 74 and 9.

> However not all the connections make as much sense as these. For example the 'Kriminalitet' (Crime) topic has two filler topics: 42 and 75, one topic about economy: 60, one topic about politics: 8, and one topic about sports: 86.

Reviewer: ```suggestion
However not all the connections make as much sense as these. 
For example, the 'Kriminalitet' (Crime) topic has two filler topics: 42 and 75, one topic about economy: 60, one topic about politics: 8, and one topic about sports: 86.
```

\subsection{Pachinko Results}\label{app:pachinko_res}

\autoref{tab:pachinko_topics} shows the final lowest level topics of the pachinko model.
Note that while most of the topics are semantically coherent, there are some topics (eg. topic 19 and 42) which consist entirely of words which provide little context or semantic meaning.
The model has also learned to group words that do not belong to any good topics.

> This is a good feature that allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics.

Author: ```suggestion
This is a good feature that allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics.
\todo[inline]{ref to hlda and possibly one newer paper (simba knew one) that does the same thing.}
```

\subsection{Pachinko Results}\label{app:pachinko_res}

\autoref{tab:pachinko_topics} shows the final lowest level topics of the pachinko model.

> Note that while most of the topics are semantically coherent, there are some topics (eg. topic 19 and 42) which consist entirely of words which provide little context or semantic meaning.

Reviewer 2: ```suggestion
Note that while most of the topics are semantically coherent, there are some topics (e.g. topic 19 and 42) which consist entirely of words which provide little context or semantic meaning.
```

The model has also learned to group words that do not belong to any good topics.
This is a good feature that allows the model to apply an extra layer of preprocessing, automatically filtering away irrelevant words into topics.

> \todo[inline]{ref to hlda and possibly one newer paper (simba knew one) that does the same thing.}

Reviewer 2: ```suggestion
\todo[inline]{ref to hlda and "Topic Modeling in Embedding Spaces" that does the same thing.}
```
Det kan ses i figur 6 i den artikel.

\begin{longtable}[c]{c | c}

> 		\caption{Top 10 words for each lowest level topic in the results of the pachinko model \label{tab:pachinko_topics}}\\

Reviewer 2: ```suggestion
		\caption{Top 10 words for each lowest level topic in the results of the pachinko model. \label{tab:pachinko_topics}}\\
```

		88 & eu, britiske, brexit, storbritannien, aftale, london, may, premierminister, johnson, theresa \\
		89 & frivillige, foreningen, løgstør, aktiviteter, lokale, foreninger, medlemmer, forening, formand, lørdag \\
		90 & landbrug, landbruget, landmænd, vand, miljø, affald, bedre, natur, vandløb, fødevarer \\
\end{longtable}

\autoref{tab:pachinko_mid_topics} gives an overview of how the taxonomy topics in the third layer of the pachinko model, are connected to the fourth layer topics that were generated by the model.
Some of these connections make a lot of sense, such as the 'Økonomi' (Economy) topic which has the three filler topics: 79, 75, and 42, which consists of words with little semantic value, and two topics which are about money: 74 and 9.
However not all the connections make as much sense as these. 
For example, the 'Kriminalitet' (Crime) topic has two filler topics: 42 and 75, one topic about economy: 60, one topic about politics: 8, and one topic about sports: 86.
See \autoref{tab:pachinko_topics}, for more details on the top words within each of these topics.

\begin{table}[H]
	\centering

> 	\caption{Ids of the top 5 most occuring fourth layer topics for each third layer topic from the pachinko model. See table \autoref{tab:pachinko_topics} for most occouring words for each id.}

Reviewer 2: ```suggestion
	\caption{IDs of the top 5 most occurring fourth layer topics for each third layer topic from the pachinko model. See \autoref{tab:pachinko_topics} for the most occurring words for each ID.}
```

	\label{tab:pachinko_mid_topics}
	\begin{tabular}{c | c | c | c | c | c}

> 		Taxonomy Name & Top 5 Topic ids & Taxonomy Name & Top 5 Topic ids & Taxonomy Name & Top 5 Topic ids \\ \hline

Reviewer 2: ```suggestion
		Taxonomy Name & Top 5 Topic IDs & Taxonomy Name & Top 5 Topic IDs & Taxonomy Name & Top 5 Topic IDs \\ \hline
```
